src/software/simulated_tests/example_play_test.cpp

```diff
@@ -31,6 +29,12 @@ TEST_F(ExamplePlayTest, test_example_play)
                     Point ball_position = world_ptr->ball().position();
                     for (const auto& robot : world_ptr->friendlyTeam().getAllRobots())
                     {
+                        // Skips the robot assigned as goalie by default in the
+                        // sensor_fusion constructor
```

Comment:
it would be better to explicitly set goalie to 0 and rely on that, rather than expecting that the default goalie is 0, which is an implementation detail

Reply:
Makes sense, I was also thinking it was a little sketchy to just assume that